collector/kernel/bpf_src/render_bpf.c

```diff
@@ -629,11 +505,6 @@ int onret_get_pid_task(struct pt_regs *ctx)
 
 int on_finish_task_switch(struct pt_regs *ctx, struct task_struct *prev)
```

Comment:
This probe will not be enabled any more, so the whole function can be removed.

Reply:
done